charts/fluent-bit/templates/_pod.tpl

```diff
@@ -13,6 +13,10 @@ securityContext:
 dnsConfig:
   {{- toYaml . | nindent 2 }}
 {{- end }}
+{{ if .Values.initContainers }}
+initContainers:
+{{ toYaml .Values.initContainers | indent 2 }}
```

Comment:
```suggestion
  {{- toYaml .Values.initContainers | nindent 2 }}
```

nit: more readable imho

Reply:
Agree with you, it's more readable.
Thanks.